gogo/gadgets.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+def init_gadget(gobase):
+    gobase.register_start()
```

Comment:
The start and stop functions are typically used to start/stop a local version of the application locally (usually a web server that a developer can access at an address like `localhost:9000`). I advise removing them here as this project is a library and cannot really be run locally in any way that I can think of at the moment. In any event, does not seem worth blocking this PR on that, but something to consider.